fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/ColumnParser.java

```diff
@@ -186,4 +192,61 @@ public boolean parse(String value) {
             throw new RuntimeException("string check failed ", e);
         }
     }
+}
+
+class DecimalParser extends ColumnParser {
+
+    public static int PRECISION = 27;
+    public static int SCALE = 9;
+
+    private BigDecimal maxValue;
+    private BigDecimal minValue;
+
+    public DecimalParser(EtlJobConfig.EtlColumn etlColumn) {
+        StringBuilder precisionStr = new StringBuilder();
+        for (int i = 0; i < etlColumn.precision; i++) {
```

Comment:
If the column definition is `k1 decimal(4,3)`, than the value range of `k1` should be 
`[-9.999, 9.999]`

And your code will give: `[-9999.999, 9999.999]`, which is not right.

Reply:
Better to refer to the logic in DecimalLiteral class.

checkPrecisionAndScale(int precision, int scale)

Reply:
👌